### DISCUSSION TITLE
ERROR `/assets/img/prof_pic.jpg' not found.

### DISCUSSION THREAD

**Discussion by anabbi** [alshedivat/al-folio#463] [Q&A] [answered]
Hi

thank you for your amazing work. For some reason, the prof_pic can not be found in /assets/img in my local. When I push and deploy, it is found, with its original dimensions, though. Can you guide me how to fix this?

Thanks
Arash

**Comment by alshedivat** [comment, marked as answer]
looks like this bug was introduced in #451. I've just reverted it in #468. please pull the latest changes from master and lmk if that worked for you. thanks!

**Reply by anabbi** [reply]
Thanks so much. The problem is now fixed. 

Just a quick note for other users that if a photo appears flipped, adjusting its aspect ratio would resolve it.

Thanks!
